doc/changes/DM-31541.feature.rst

```diff
@@ -0,0 +1 @@
+Make pipetask responsible for initalization read only select nodes from the quantum graph.
```

Comment:
Difficult to read.  Maybe something like:  Create list of node ids for the pipetask --init-only job.